### ISSUE TITLE
Stylesheet at #deletelink

### ISSUE THREAD

**Issue by ukcb** [PrivateBin/PrivateBin#266]
Please make a small change in the stylesheet for better usability:

<pre>
#deletelink {
    float: right;
    background-color: #ffa;
    border: 3px solid #f00;
    margin-left: 10em;
    padding: .3em;
}
</pre>

Or something like that, the main thing you see the delete link better.
Many Thanks.

**Comment by rugk** [comment]
Thanks for your suggestion. You can send us a [PR](https://help.github.com/articles/creating-a-pull-request/) for that. Just edit PrivateBin's [bootstrap CSS file](https://github.com/PrivateBin/PrivateBin/blob/master/css/bootstrap/privatebin.css#L77).

**Comment by rugk** [comment]
You do not need to get deep into that at all, and you can easily edit the file online at GitHub e.g. So it is really easy and fast to do! 😃 

If you also change it in your browser, you can make a screenshot and then we can look whether we like the change, you propose.

**Comment by ukcb** [comment]
Ok, I'll try it.

**Comment by ukcb** [comment]
I tried, hopefully I did not do anything wrong. :-)

**Comment by rugk** [comment]
No, you did not do anything wrong with the PR. It worked all right. We are just reviewing it, and may request some changes and so on. You can just commit & push these changes to the same branch as you used for the PR and they appear on GitHub automatically.

**Comment by ukcb** [comment]
I do not really care about the colors. I am concerned with the delimitation of the other text, therefore the right distance. The problem should be clear in the screenshot. I do not think the colors are that bad. :-)

![fertig](https://user-images.githubusercontent.com/22983914/34467157-444b0518-eee9-11e7-9a57-44c3b87c2f88.png)

**Comment by Ribas160** [comment]
Hi everyone. I see that we already have a pull request here. I just want to share my idea with you. Not so far ago we were working on the button for copying the link to the clipboard #1475. What do you think about implementing a similar solution here: the button above the link message? I think it would be more mobile-friendly than a button inside the message, because the link is quite long and can span multiple lines, especially if there's a button inside the same message.

Here are examples how it could looks like:

On desktop version
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/ad710a50-d505-43f7-9511-ddb7ca2ba6b7" />

On mobile version
![localhost__781dbf38183c1535(iPhone XR)](https://github.com/user-attachments/assets/69d7301e-97ba-4e6a-83a6-46264683e7ba)

The only thing I am thinking about, isn't here too much buttons? I am not a designer and it is difficult for me to unserstand is it looks good or not?

**Comment by elrido** [comment]
I do like the buttons as suggested in these screenshots. To avoid having to write and maintain too much custom CSS, we could use a [Nav (not Navbar) from Bootstrap 5](https://getbootstrap.com/docs/5.3/components/navs-tabs/#horizontal-alignment) (see also [Bootstrap 3 docs](https://getbootstrap.com/docs/3.4/components/#nav)).

We could use a right aligned one, for the delete link to stay on the right, or a [filled](https://getbootstrap.com/docs/5.3/components/navs-tabs/#fill-and-justify) and [justified](https://getbootstrap.com/docs/3.4/components/#nav-justified) one?
